tests/app/Infrastructure/Controller/GetUsersListControllerTest.php

```diff
@@ -27,16 +27,54 @@ protected function setUp(): void
     /**
      * @test
      */
-    public function noUsersFound()
+    public function noListedUsersFound()
     {
         $this->userDataSource
-            ->expects('listAll')
-            ->never();
-            //->once();
+            ->expects('listedUsers')
+            ->once();
 
         $response = $this->get("/api/users/list");
 
         $response->assertExactJson([]);
```

Comment:
Here is necessary a code status verification, add please.

Reply:
Resolved